ftp/sftp/README.md

```diff
@@ -0,0 +1,58 @@
+# Setup
+
+```
+git clone https://github.com/shipyardapp/starter-blueprints.git
+cd starter-blueprints/
+python3.7 -m venv venv
+venv/bin/python setup.py install
+```
+
+You will need the sftp server hostname/address as well as port if it's not the
+default sftp port(21) as well as the user credentials
```

Comment:
Default port for SFTP is 22, no?

Reply:
Fixed, this probably happened when I copied the ftp README and forgot to change this portion of the README.